3scale/porta
# THREESCALE-8878: Automate managed services builds

**What this PR does / why we need it**:

Modify the `Fast-forward dev or stable branch to a SHA1` GitHub action to fast-forward from `master` to `managed-services`

**Which issue(s) this PR fixes** 

https://issues.redhat.com/browse/THREESCALE-8878

**Verification steps** 

Wait until Novermber 17th at 7:00 UTC and check that `managed-services` has been fast-forwarded to `master`
